docs/source/development/index.rst

```diff
@@ -0,0 +1,56 @@
+👩‍💻 Development
+=================
+
+Quick start
+-----------
+
+To obtain a copy of the code and work on a new branch::
+
+    git clone https://github.com/NASA-PDS/data-upload-manager.git
+    git checkout -b "<issue number>_<issue name>"
+
+Create a virtual environment in ``venv`` using Python 3.9 or later::
+
+    python3 -m venv venv
+
+Install the package and its dependencies for development into the virtual environment::
+
+    pip install --editable '.[dev]'
+
+At this point, the command line tools are now available in ``venv/bin``
+as explained in `usage`_.
+
+Testing
+-------
+
+The code base includes unit tests. Once you've installed the service, you can
+run the unit tests with the following command:
+
+    venv/bin/tox py39
+
+
+Making Releases
+---------------
+
+Releases are done on GitHub and PyPI. This is implemented through the CI/CD
+framework using GitHub Actions_ on the ``main`` branch. To trigger a release
+via the CI/CD framework, create a branch named ``release/<version number>``,
```

Comment:
We haven't triggered releases via branch in a while. If this text is coming from the template repo, then it needs to be fixed too.

To trigger a release now, do:

    git tag --annotate --message "Release 1.2.0" release/1.2.0
    git push origin release/1.2.0

And it must be a fully-qualified X.Y.Z version.